### PR TITLE
Add pyproject.toml for Custom Node Registry

### PR DESCRIPTION
**Pull request by haohaocreates**
We are working with dr.lt.data and comfyanon to build a global registry for custom nodes (similar to PyPI). Eventually, the registry will be used as a backend for the UI-manager. All nodes go through a verification process before being published to users.

The main benefits are that authors can
- publish nodes by version and users can safely update nodes knowing ahead of time if their workflows will break or not
- automate testing against new commits in the comfy repo and existing workflows through our CI/CD dashboard

Action Required:

- [ ] Go to the [registry](https://registry.comfy.org). Login and create a publisher id (everything after the `@` sign on your registry profile). 
- [ ] Add the publisher id into the pyproject.toml file.
- [ ] Merge the separate Github Actions PR, then merge this PR.

If you want to publish the node manually, [install the cli](https://docs.comfy.org/comfy-cli/getting-started#install-cli) by running `pip install comfy-cli`, then run `comfy node publish`

Check out our [docs](https://docs.comfy.org/registry/overview#introduction) if you want to know more about the registry. Otherwise, feel free to message me on discord at haohao_81202 or join our [server](https://discord.com/invite/comfyorg) if you have any questions!

### CODE REVIEW

**Comment by haohaocreates** [comment]
hey @Danand, wanted to send a small nudge and ask if you had any questions!

**Comment by haohaocreates** [comment]
@Danand, wondering if there's any way we can support you on this PR!

**Comment by Danand** [comment]
@haohaocreates 
Hi! Sorry for the delayed reply. I will check it out soon. Thank you!

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[project]
+name = "comfyui-comfycouple"
+description = " If you want to draw two different characters together without blending their features, so you could try to check out this custom node."
+version = "1.0.0"
+license = "LICENSE"
```
Comment (Danand) [inline pyproject.toml:5]:
```suggestion
license = "GPL-3.0"
```

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+[project.urls]
+Repository = "https://github.com/Danand/ComfyUI-ComfyCouple"
+#  Used by Comfy Registry https://comfyregistry.org
+
+[tool.comfy]
+PublisherId = ""
```
Comment (Danand) [inline pyproject.toml:12]:
```suggestion
PublisherId = "rei---"
```

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+[project]
+name = "comfyui-comfycouple"
+description = " If you want to draw two different characters together without blending their features, so you could try to check out this custom node."
+version = "1.0.0"
+license = "LICENSE"
+
+[project.urls]
+Repository = "https://github.com/Danand/ComfyUI-ComfyCouple"
+#  Used by Comfy Registry https://comfyregistry.org
```
Comment (Danand) [inline pyproject.toml:9]:
```suggestion
```

**Review by Danand — CHANGES_REQUESTED** [review]

**Review by Danand — APPROVED** [review]

**Pull request merged**